defaults/adv_settings.yml.default

```diff
@@ -22,9 +22,12 @@ traefik:
     jaeger: jaeger
   error_pages: no
 mounts:
-  remote: rclone_vfs
+  remote: rclone_vfs # or rclone_vfs_cache
```

Comment:
We do not want comments in the configuration files as they will be removed when configuration files are upgraded anyway.